libs/vertexai/langchain_google_vertexai/embeddings.py

```diff
@@ -470,27 +470,31 @@ def embed_query(self, text: str) -> List[float]:
 
     def embed_image(
         self,
-        image_path: str,
+        uris: List[str],
```

Comment:
it seems like a breaking change to me, can we add a deprecation warning instead and implement a new method maybe?

Reply:
added deprecation warning and new method